config/initializers/docker.rb

```diff
@@ -0,0 +1,22 @@
+require 'tempfile'
+tmp_dir = "#{Rails.root}/tmp"
+cert_file = Tempfile.new "docker_cert", tmp_dir 
+cert_file.write(ENV["DOCKER_CERT"])
+cert_file.close
+
+key_file = Tempfile.new "docker_key", tmp_dir 
+key_file.write(ENV["DOCKER_KEY"])
+key_file.close
+
+ca_file = Tempfile.new "docker_ca", tmp_dir 
+ca_file.write(ENV["DOCKER_CA"])
+ca_file.close
+
+DOCKER_KEYS=[ca_file, key_file, cert_file]
```

Comment:
This is done to prevent the keys from being gc'd, any ideas on alternatives for this?